src/funding/card/config.jsx

```diff
@@ -90,85 +82,94 @@ export function getCardConfig() : FundingSourceConfig {
+                                color: #000;
+                                blackground: #fff;
+                            }
+
+                            .paypal-logo-color-white #black-credit-card-button { 
+                                color: #fff;
+                            }
+
+                            #black-credit-card-button {
+                                background: transparent;
+                                font-size: 1.15em;
+                            }
+
+                            #black-credit-card-button img {
+                                display: inline-block;
+                                height: 1.3em;
+                                vertical-align: middle;
+                                padding: 0 9px;
+                            }
+
+                            .dom-ready #black-credit-card-button div {
+                                -webkit-animation: show-text 1.0s 0s forwards;
+                                -moz-animation:    show-text 1.0s 0s forwards;
+                                -o-animation:      show-text 1.0s 0s forwards;
+                                animation:         show-text 1.0s 0s forwards;
+                                vertical-align: middle;
+                                overflow: hidden;
+                                display: inline-block;
+                                max-width: 0%;
+                                opacity: 0;
+                                position: static;
+                                visibility: visible;
+                            }
+
+                            #black-credit-card-button div {
+                                position: absolute;
+                                visibility: hidden;
+                            }
+                        ` } />
+                    <button
+                        type="button"
+                        id="black-credit-card-button"
+                        onClick={ event => onClick(event) }
+                        class={ `${ CLASS.BUTTON }` }
                         { ...{
-                            [ ATTRIBUTE.FUNDING_SOURCE ]: FUNDING.CARD,
-                            [ ATTRIBUTE.CARD ]:           name
-                        } }
-                    >
-                        <Label />
-                    </div>
-                );
-            }).filter(Boolean).slice(0, maxCards);
+                            [ ATTRIBUTE.FUNDING_SOURCE ]: FUNDING.CARD
+                        } }>
+                        { isRTLLanguage && (<div>{ content.payWithDebitOrCreditCard }</div>) }
+                        <GlyphCard color={ logoColor === LOGO_COLOR.BLACK ? '#000000' : '#ffffff' } />
+                        { !isRTLLanguage && (<div>{ content.payWithDebitOrCreditCard }</div>) }
+                    </button>
+                </div>
+            );
         },
 
-        Label: ({ logo }) => logo,
+        Label: ({ logo }) => {
+            return (
```

Comment:
To follow convention, can we have:

1. `Logo` function should return the card logo glyph
2. `Label` function should return `{ logo } <Text>{ content.payWithDebitOrCreditCard }</Text>`